CONTRIBUTING.md

```diff
@@ -124,6 +124,7 @@ $ TEST_DEBUG=true make test
 To test the code coverage, use:
 
 ```sh
+$ NODE_ENV=test BABEL_ENV=cov make build
```

Comment:
I don't think we need `NODE_ENV=test` here because it is exported by default, see https://github.com/babel/babel/blob/master/Makefile#L3.